scripts/pi-hole/js/queries.js

```diff
@@ -115,6 +115,12 @@ $(function () {
         isCNAME = false,
         regexLink = false;
 
+      // accompanies Store domainlist IDs for blocked/permitted queries FTL PR 1409
+      // 'is_numeric' is undefined, replaced with '!isNAN', see PR for details
+      if (data.length > 9 && !isNaN(data[9]) && data[9] > 0) {
```

Comment:
I don't like double negatives.
```suggestion
      if (data.length > 9 && Number.isInteger(data[9]) && data[9] > 0) {
```

Reply:
I'm sorry, I did test this, since it was suggested in one of the duckduckgo results a linked (above), however, it doesn't work, tested with firefox and edge chromium)

result (no link) with if (data.length > 9 && Number.isInteger(data[9]) && data[9] > 0) {:

![image](https://user-images.githubusercontent.com/24669386/189331447-f7b285ac-d36f-4670-8dc4-b4a5a00871bc.png)

result (link) with if (data.length > 9 && !isNaN(data[9]) && data[9] > 0) {

![image](https://user-images.githubusercontent.com/24669386/189331672-c5bb89e9-3ede-470b-b0b2-0ae4d18a9d78.png)

Reply:
Try `Number.isInteger(parseInt(data[9],10))`

Reply:
That works, but again, aren't we over complicating things? Remember that everything worked fine without the numeric test. we now add 'Number.isInteger' and 'parseInt(data[9]) for every query (all status types)...